packages/components/text/src/text.styles.ts

```diff
@@ -56,15 +56,49 @@ const getTone = (tone: string) => {
 const getElementFontSize = (elementType?: string) => {
   switch (elementType) {
     case 'h1':
-      return '2.4615rem';
+      return designTokens.fontSizeForTextAsH1;
     case 'h2':
-      return '1.8462rem';
+      return designTokens.fontSizeForTextAsH2;
     case 'h3':
-      return '1.5385rem';
+      return designTokens.fontSizeForTextAsH3;
     case 'h4':
-      return '1.2308rem';
+      return designTokens.fontSizeForTextAsH4;
     case 'h5':
-      return '1.0769rem';
+      return designTokens.fontSizeForTextAsH5;
+    default:
+      return '';
```

Comment:
Shouldn't we return a value here? 🤔 (same in the other functions)

Reply:
Could we `return elementType`; since we don't have default font size for all of them?
And for the tone return `designTokens.colorPrimary;`?

Reply:
> Could we `return elementType;` since we don't have default font size for all of them?

Maybe we can return `1rem`?

> And for the tone return `designTokens.colorPrimary;`?

For the tone is fine to return an empty string, because the other values return the full style `color: ???`.

For the `font-size`, `line-height`, `font-weight` the function returns the actual value (not the full style), so I think it would be better to return a value rather than having potentially an empty style (e.g. `font-size: ;`).

What do you think?

Reply:
Ah, yes I get your point and I agree. Thank you!